schema/test/unit/mutations/undo_form_changes_test.sql

```diff
@@ -124,9 +124,9 @@ select is(
     select new_form_data from cif.form_change
     where id = 3
   ),
-  null::jsonb
+  jsonb_build_object()
```

Comment:
This would make sure that we don't depend on the `jsonb_build_object` implementation - and improve readability
```suggestion
  '{}'::jsonb
```